config-linux.md

```diff
@@ -8,7 +8,7 @@ The Linux container specification uses various kernel features like namespaces,
 The Linux ABI includes both syscalls and several special file paths.
 Applications expecting a Linux environment will very likely expect these files paths to be setup correctly.
 
-The following filesystems MUST be made available in each application's filesystem
+The following filesystems MUST be made available in each containerized application's filesystem
```

Comment:
@crosbmichael pointed out a while back that [we do not define “application”][1].  Can we use “… in the container mount namespace”?  The glossary does define “[container namespace][2]”.

[1]: https://github.com/opencontainers/runtime-spec/pull/107#issuecomment-145615479
[2]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc2/glossary.md#container-namespace